public/app/core/components/sidemenu/SideMenuItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ReactNode } from 'react';
+import SideMenuDropDown from './SideMenuDropDown';
+import { Link, useStyles2 } from '@grafana/ui';
+import { NavModelItem } from '@grafana/data';
+import { css, cx } from '@emotion/css';
+
+export interface Props {
+  children: ReactNode;
+  label: string;
+  menuItems?: NavModelItem[];
+  menuSubTitle?: string;
+  onClick?: () => void;
+  reverseMenuDirection?: boolean;
+  target?: HTMLAnchorElement['target'];
+  url?: string;
+}
+
+const SideMenuItem = ({
+  children,
+  label,
+  menuItems = [],
+  menuSubTitle,
+  onClick,
+  reverseMenuDirection = false,
+  target,
+  url,
+}: Props) => {
+  const resetButtonStyles = useStyles2(
+    () =>
+      css`
+        background-color: transparent;
+      `
+  );
+
+  const anchor = url ? (
+    <Link
+      className="sidemenu-link"
+      href={url}
+      target={target}
+      aria-label={label}
+      onClick={onClick}
```

Comment:
❓ out of curiosity: I think this is part of the old code, but do you know in which cases we pass a URL and onClick at the same time? I found it a bit confusing 😕

Reply:
yeah i don't think any part of the old code does it. 😄  it either uses `onClick` or passes a url. i can imagine _some_ scenarios where you'd want both, so left the `onClick` on there as well. technically it's valid. can remove if we think it's confusing, i don't have a strong opinion tbh 🤷

Reply:
I also don't have a strong opinion, for only curiosity 😄 , so let's keep it, we might need it in the future